src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs

```diff
@@ -519,6 +520,10 @@ private static object ProcessValue(object obj, int currentDepth, in ConvertToJso
             {
                 rv = jObject.ToObject<Dictionary<object, object>>();
             }
+            else if (obj is BigInteger bi)
+            {
+                rv = new JRaw(bi.ToString(NumberFormatInfo.InvariantInfo));
```

Comment:
Why do we need `JRaw`?

Reply:
So the value isn’t quoted and is a literal numeric value.

Reply:
Perhaps we should use `string` if we agree with https://github.com/PowerShell/PowerShell/pull/20999#issuecomment-1878187217

Reply:
Personally I don't see why we should but I'll leave it up to the WG decision in the end.

Reply:
Turns out it isn't needed we just need to ensure the value is the BigInteger itself and we don't do PowerShell's property enumerator.